jupyterhub/tests/test_browser.py

```diff
@@ -0,0 +1,41 @@
+import asyncio
```

Comment:
I accidentally added this file. Let's remove it.

Reply:
done